unfoldingWord/translationCore
# Now Resetting the online alert message when user logs out

#### This pull request addresses:

- Now Resetting the 'do not show this warning again' flag when the user logs out
 #2181


#### How to test this pull request:
- Log in with one user try to access the internet. An alert should show up. Click the 'Don't show again' check box.
- Now l log out from that user and log in as a different user and try to access the internet. the alert should show up even though you clicked 'don't show again' with the other user.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/2500)
